tests/create-release.test.js

```diff
@@ -321,7 +353,8 @@ describe('Create Release', () => {
       name: 'myRelease',
       body: '',
       draft: false,
-      prerelease: false
+      prerelease: false,
+      target_commitish: 'sha-value'
```

Comment:
I can now see a lot of changes in the test because we introduced one more optional property. Is this the nature of `Jest` or could we just add one more test checking if `target_commitish` was set?